jpa/eclipselink.jpa.test/pom.xml

```diff
@@ -184,740 +178,36 @@
-                            <outputFile>${project.build.directory}/generated-test-sources/jpa-runner-src/org/eclipse/persistence/testing/framework/jpa/server/TestRunnerBean.java</outputFile>
-                            <replacements>
-                                <replacement>
-                                    <token>@puName@</token>
-                                    <value>${puName}</value>
-                                </replacement>
-                            </replacements>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-
-            <!--Generate some classes required for test by EclipseLink Canonical Model Processor.
-                Adds to the compiler (test scope) sources root path: .../target/generated-test-sources -->
-            <plugin>
-                <groupId>org.bsc.maven</groupId>
-                <artifactId>maven-processor-plugin</artifactId>
-                <configuration>
-                    <processors>
-                        <processor>org.eclipse.persistence.internal.jpa.modelgen.CanonicalModelProcessor</processor>
-                    </processors>
-                    <sourceDirectory>${project.basedir}/${integration.test.sources.directory}</sourceDirectory>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>generate-canonical-meta-model-classes</id>
-                        <goals>
-                            <goal>process-test</goal>
-                        </goals>
-                        <phase>generate-test-sources</phase>
-                        <configuration>
-                            <outputDirectory>${project.build.directory}/generated-test-sources/canonical-meta-model-src</outputDirectory>
-                            <compilerArguments>-Aeclipselink.persistencexml=${project.basedir}/${integration.test.resources.directory}/eclipselink-canonical-meta-model/persistence.xml
-                            </compilerArguments>
-                            <includes>
-                                <include>org/eclipse/persistence/testing/models/jpa/advanced/*.java</include>
-                            </includes>
-                        </configuration>
-                    </execution>
-                </executions>
-                <dependencies>
-                    <dependency>
-                        <groupId>org.eclipse.persistence</groupId>
-                        <artifactId>org.eclipse.persistence.jpa.modelgen.processor</artifactId>
-                        <version>${project.version}</version>
-                    </dependency>
-                </dependencies>
-            </plugin>
-            <!--Generate multiple jar files required for tests.-->
-            <plugin>
-                <groupId>com.soebes.maven.plugins</groupId>
-                <artifactId>iterator-maven-plugin</artifactId>
```

Comment:
Maybe after this change `iterator-maven-plugin` should be removed from whole project.

Reply:
there's more cleanup needed, therefore I prefer to postpone this to the other/followup PR